apps/system/utils/netcmd.c

```diff
@@ -206,6 +206,25 @@ int cmd_ifdown(int argc, char **argv)
 	return ret;
 }
 
+int cmd_netmon(int argc, char **argv)
+{
+	FAR char *intf = NULL;
+	int ret;
+
+	if (argc > 2) {
+		printf("Bad input\n");
+		return OK;
+	}
+	if (argc == 1) {
+		intf = "all\n";
+	} else {
+		intf = argv[1];
+	}
+	ret = netlib_netmon(intf);
```

Comment:
It using ret only here, this is one of reason of warning.
you can just return netlib_netmon here.
Also please remove annotated printf below.

Reply:
I agree to your comments. Modification completed.